packages/syft/src/syft/core/node/domain/client.py

```diff
@@ -719,74 +725,3 @@ def load_dataset(
         print(
             "\n\nRun <your client variable>.datasets to see your new dataset loaded into your machine!"
         )
-
-    def create_dataset(
```

Comment:
Delete this method, cause I do not feel the need to create empty datasets anymore, since `load_dataset` is now very fast.

Reply:
I guess the `CreateDatasetMessage: create_dataset_msg,` on the `dataset_manager_service` side can be left for now until we refactor the services?